packages/salesforcedx-vscode-core/src/util/configUtil.ts

```diff
@@ -22,19 +26,47 @@ export enum ConfigSource {
   None
 }
 
-// This class should be reworked or removed once the ConfigAggregator correctly checks
-// local as well as global configs. It's also worth noting that ConfigAggregator, according
-// to its docs checks local, global and environment and, for our purposes, environment may
-// not be viable.
+async function getConfigAggregator(): Promise<ConfigAggregator> {
+  const origCurrentWorkingDirectory = process.cwd();
+  const rootWorkspacePath = getRootWorkspacePath();
+  // Change the current working directory to the project path,
+  // so that ConfigAggregator reads the local project values
+  process.chdir(rootWorkspacePath);
+  const configAggregator = await ConfigAggregator.create();
+  // Change the current working directory back to what it was
+  // before returning
+  process.chdir(origCurrentWorkingDirectory);
+  return configAggregator;
+}
```

Comment:
When changing the current working directory, I suggest wrapping the code in a try/finally block, so that in case anything throws, the working directory is set back.  Something like this:
```
async function getConfigAggregator(): Promise<ConfigAggregator> {
  const origCurrentWorkingDirectory = process.cwd();
  const rootWorkspacePath = getRootWorkspacePath();
  try {
    // Change the current working directory to the project path,
    // so that ConfigAggregator reads the local project values
    process.chdir(rootWorkspacePath);
    const configAggregator = await ConfigAggregator.create();
  } finally {
    // Change the current working directory back to what it was
    // before returning
    process.chdir(origCurrentWorkingDirectory);
  }    
  return configAggregator;
}
```
(warning: untested code)



Also, does the CWD need to be changed every time `ConfigAggregator.create()` is called?  If so, maybe move the CWD code into `ConfigAggregator.create()`.